tests/test_tag_parser.py

```diff
@@ -9,96 +9,178 @@
 class TagParserTests(BaseTestCase):
     def test_tag_parser(self):
         _, attrs = parse_tag_attrs("component 'my_comp' key=val key2='val2 two' ")
+
+        expected_attrs = [
+            TagAttr(key=None, value="component", start_index=0, quoted=None, spread=False, translation=False),
+            TagAttr(key=None, value="my_comp", start_index=10, quoted="'", spread=False, translation=False),
+            TagAttr(key="key", value="val", start_index=20, quoted=None, spread=False, translation=False),
+            TagAttr(key="key2", value="val2 two", start_index=28, quoted="'", spread=False, translation=False),
+        ]
+
+        self.assertEqual(attrs, expected_attrs)
         self.assertEqual(
-            attrs,
+            [a.formatted() for a in attrs],
```

Comment:
Updated the tests to also check that we get the original formatting once again when we call `TagAttr.formatted()`